src/main/steps/applicant1/provide-information-to-the-court/template.njk

```diff
@@ -0,0 +1,181 @@
+        <form class="form" id="main-form" method="post" action="" novalidate="novalidate">
+          <input type="hidden" name="_csrf" id="csrfToken" value="{{ csrfToken }}">
+          {% set textArea %}
+            {% set fieldName = 'coClarificationResponses' %}
+            {% set field = form.fields[fieldName] %}
+            {% include "common/form/fields/" + field.type + ".njk" %}
+          {% endset %}
+          {{ textArea | safe }}
+          <h4 class="govuk-heading-s">{{ uploadTitle }}</h4>
+          <p class="govuk-body">{{ line5 }}</p>
+          {{ govukWarningText({
+            text: warningPhoto,
+            iconFallbackText: "Warning"
+          }) }}
+          {% set infoTakePhotoDetails %}
+            <ol class="govuk-list govuk-list--number">
+              <li>{{ infoBullet1 }}</li>
+              <li>{{ infoBullet2 }}</li>
+              <li>{{ infoBullet3 }}</li>
+              <li>{{ infoBullet4 }}</li>
+              <li>{{ infoBullet5 }}</li>
+            </ol>
+          {% endset %}
+          {{ govukDetails({
+            summaryText: infoTakePhoto,
+            html: infoTakePhotoDetails
+          }) }}
+          <p class="govuk-body">{{ minRequirements }}</p>
+
+          <script id="uploadContent" type="application/json">
+          {
+            "isClarificationAmendableState": {{ isClarificationAmendableState }},
+            "delete": "{{ delete }}"
+          }
+          </script>
+
+          <div id="uploadGroup" tabindex="0" class="uploadGroup">
+            <h2 class="govuk-heading-s">{{ uploadFiles }}</h2>
+            <p class="govuk-body govuk-section-break govuk-section-break--visible govuk-!-padding-bottom-3{% if userCase.coClarificationUploadDocuments|length %} hidden{% endif %}" id="noFilesUploaded">{{ noFilesUploaded }}</p>
+            <ul id="filesUploaded" class="govuk-list">
+              {% for file in userCase.coClarificationUploadDocuments %}
+                <li class="uploadedFile govuk-!-padding-top-2 govuk-!-padding-bottom-3 govuk-section-break govuk-section-break--visible">
+                  {{ file.value.documentFileName }} <a href="/document-manager/delete/{{ loop.index0 }}?_csrf={{ csrfToken }}" class="govuk-link--no-visited-state">{{ delete }}</a>
+                </li>
+              {% endfor %}
+            </ul>
+          </div>
+
+          <p class="govuk-body">
+            <strong>{{ acceptedFileFormats }}</strong> {{ fileFormats }}<br>
+            <strong>{{ maximumFileSize }}</strong> {{ fileSize }}
+          </p>
+
+          <div class="govuk-form-group {{ 'hidden' if not isClarificationAmendableState }}">
+            <div class="govuk-panel upload hidden">
+              <span class="upload-arrow-icon govuk-!-margin-bottom-5"></span>
+              <div class="govuk-form-group">
+                <p class="govuk-body govuk-!-margin-bottom-0">
+                  <span id="upload"></span>
+                  <strong>{{ orStr }}</strong><br>
```

Comment:
what's orStr, can't see it anywhere else?

Reply:
just the string 'or', its in the upload content file